$OpenBSD: patch-prnt_hpcups_CommonDefinitions_h,v 1.2 2016/09/19 09:17:45 ajacoutot Exp $

Fix build with cups>=2.2.0

Remove prehistorical header which is not even used.

--- prnt/hpcups/CommonDefinitions.h.orig	Fri Aug 26 12:03:22 2016
+++ prnt/hpcups/CommonDefinitions.h	Sun Sep 18 19:31:21 2016
@@ -40,6 +40,7 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 #include <cups/cups.h>
+#include <cups/ppd.h>
 #include <cups/raster.h>
 #include <stdarg.h>
 #include <stdio.h>
@@ -48,7 +49,6 @@
 #include <syslog.h>
 #include <assert.h>
 #include <time.h>
-#include <sys/timeb.h>
 #include <string.h>
 
 #ifndef _GNU_SOURCE
